packages/core/examples/tds/invalid/typoCheckWithTypos.json

```diff
@@ -0,0 +1,46 @@
+{
+    "id2": "urn:typoCheck",
+    "x@context":"https://www.w3.org/2022/wot/td/v1.1",
```

Comment:
How would JSON Schema detect @context typo? Wouldn't it be considered as an 'additional property' added intentionally by user?

Reply:
So it should be removed from tests I think. I think there is no need to add extra code to not check mandatory fields. The typo detection is working better than JSON Schema validation here since it actually provides guidance to users